engine/texc/src/wscript

```diff
@@ -23,4 +30,22 @@ def build(bld):
 
     bld.install_files('${PREFIX}/include/texc', 'texc.h')
 
-    bld.recurse('test')
+
+    classpath = [os.path.join(bld.env.DYNAMO_HOME, '../../com.dynamo.cr/com.dynamo.cr.bob/bin/lib/jna-5.10.0.jar'),
+                 os.path.join(bld.env.DYNAMO_HOME, '../..//com.dynamo.cr/com.dynamo.cr.bob/bin/lib/jna-platform-5.10.0.jar')]
```

Comment:
Not a huge deal but It would be nice to not have to specify the exact version here in case we start updating the jna jars

Reply:
I think it's better to move to JNI instead tbh